### PR TITLE
mike

### PR DESCRIPTION
**Pull request by ranerlich7**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,4 +2,6 @@
 <br>
 how are you
 <br>
```
Comment (yahalierlich) [inline index.html:4]: why did you add this comment?

Comment (ranerlich7) [reply index.html:4]: i did not mean to
